### ISSUE TITLE
Elevation doesn't work when inside a stack navigator

### ISSUE THREAD

**Issue by aarjithn** [react-native-community/react-native-tab-view#553]
### Current behaviour

Tab view header is coming under the stack navigator header

### Expected behaviour

Tab view header should come in the same level of stack navigator header

### Code sample

https://snack.expo.io/@aarjithn/tabview-with-stack-navigator

### Screenshots (if applicable)

![screenshot_20180702-114541_expo](https://user-images.githubusercontent.com/10304259/42148088-70d86136-7dee-11e8-8d07-a714dd02642f.jpg)

### What have you tried

Tried to set static appbarElevation, but that doesn't help in raising the tabbar header 

### Your Environment

| software              | version
| --------------------- | -------
| ios or android        | android
| react-native-tab-view | 1.0.2

**Comment by userbq201** [comment]
@aarjithn  maybe use marginTop: -5 ?

**Comment by userbq201** [comment]
@aarjithn lke this
```js
            <View style={{ backgroundColor: 'yellow', marginTop: -5 }}>
              <TabBar {...props} style={{  backgroundColor: 'red', marginBottom: 10, paddingTop: 5 }}/>
            </View>
```

**Comment by userbq201** [comment]
@aarjithn hm. Margin doesn't fix this bag. Set `backgroundColor: 'transparent' ` to View wrapper

**Comment by satya164** [comment]
Hey, I just released a new alpha `2.0.0-alpha.0` of the library. It's rewritten using [react-native-gesture-handler](https://github.com/kmagiera/react-native-gesture-handler) and [react-native-reanimated](https://github.com/kmagiera/react-native-reanimated) addresses a many platform specific bugs and performance problems. The documentation is updated as well.

Please try the new version and see if it addresses your issue. If not, please open a new issue following the issue template.
